mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksView.kt

```diff
@@ -295,4 +306,16 @@ interface MavericksView : LifecycleOwner {
         onFail: (suspend (Throwable) -> Unit)? = null,
         onSuccess: (suspend (T) -> Unit)? = null
     ) = _internalSF(subscriptionLifecycleOwner, asyncProp, deliveryMode, onFail, onSuccess)
+
+    fun <T : Any> collectLatest(
+        flow: Flow<T>,
+        deliveryMode: DeliveryMode,
+        action: suspend (T) -> Unit
+    ): Job = flow.collectLatest(
+        subscriptionLifecycleOwner,
+        mavericksViewSubscriptionViewModel.lastDeliveredStates,
```

Comment:
this does `ViewModelProvider(this).get(MavericksViewSubscriptionViewModel::class.java)` twice - better to store in a local property to optimize.